Rewrite README and startup.sh for GOV.UK consistency (#955)
alphagov/service-manual-frontend: README.md

# Service Manual Frontend

> A public-facing GOV.UK application used to display the [service manual](https://www.gov.uk/service-manual) using content is fetched from the [Content Store](https://github.com/alphagov/content-store).

Reviewer: ```suggestion
A public-facing GOV.UK application used to display the [service manual](https://www.gov.uk/service-manual) with content fetched from the [Content Store](https://github.com/alphagov/content-store).
```

Author: Oh my, how did I miss that 🙈  - thanks.
